controllers/view.py

```diff
@@ -33,20 +33,17 @@ def is_user_subscribed(self, stream_id):
         user = users.get_current_user()
         stream = Stream.get_by_id(stream_id)
         return ConnexusUser.is_subscribed(user.user_id(), stream.key)
-        
 
     def get(self):
         user = users.get_current_user()
         stream_id = long(self.request.get('id'))
+        limit = int(self.request.get('limit', '10'))
         logout_url = users.create_logout_url('/')
 
         if not stream_id :
             self.display_error('The stream you are trying to access does not exist. It may have been removed by the owner.')
         else : 
-            view_api_uri = '{}?id={}'.format(self.uri_for('api-view', _full=True), stream_id)
-            stream_data = { 'id': stream_id }
-            json_data = json.dumps(stream_data);
-
+            view_api_uri = '{}?id={}&offset=0&limit={}'.format(self.uri_for('api-view', _full=True), stream_id, limit)
```

Comment:
Is the offset always 0? Meaning the pagination is more like a "view more"?

Reply:
Yeah is a view more, that's what I see in the requirements. I left the offset so that JS later can take advantage of it.

Reply:
Gotcha.. I see the changes in the markup to reflect this behavior too.